code/game/machinery/doors/airlock.dm

```diff
@@ -983,6 +983,15 @@ About the new airlock wires panel:
 		if(note)
 			to_chat(user, "<span class='warning'>There's already something pinned to this airlock! Use wirecutters or your hands to remove it.</span>")
 			return
+
+		if(!user.canUnEquip(C))
+			to_chat(user, "<span class='warning'>For some reason, you can't attach [C]!</span>")
```

Comment:
I hate nodrop ifs

Reply:
It's like a piece of sticky paper that won't come off your hand.